frontend/src/components/TaskCreator/index.tsx

```diff
@@ -75,21 +75,16 @@ export class TaskCreator extends React.PureComponent<Props, State> {
 
   private addTask: HTMLInputElement | null | undefined;
 
-  private darkModeStyle: CSSProperties;
-
-  constructor(props: Props) {
-    super(props);
-    this.darkModeStyle = {
-      background: 'black',
-      color: 'white',
-    };
-  }
-
   private get isOpen(): boolean {
     // eslint-disable-next-line react/destructuring-assignment
     return this.state.opened || this.props.taskCreatorOpened || false;
```

Comment:
Why is `false` needed here? I don't think `false` affects what value is returned.

Reply:
to make sure it type checks